app/_includes/about.html

```diff
@@ -82,6 +82,20 @@ <h1 class="title feature-header">{{site.data[locale].about.title}}</h1>
 				<a href="mailto:partnerships@hotosm.org" class="btn btn-blue">{{site.data[locale].about.contact_btn}}</a>
 			</div>
 
+			<div class="center-text panel">
+					<div class="title section-header">{{site.data[locale].about.communities.title}}</div>
+				</div>
+
+				<div class="about-textbox">
+					<p style="text-align: left">{{site.data[locale].about.communities.text1}}</p>
+					<p style="text-align: left">{{site.data[locale].about.communities.text2}}</p>
+				</div>
+			</div>
+
+			<div class="about-contact center-text">
+				<a href="https://drive.google.com/drive/folders/1U80TG2elfc9JNnLoKVmbU1mDcK4zCaw6?usp=share_link" class="btn btn-blue">{{site.data[locale].about.reach_btn}}</a>
+			</div>
+			
```

Comment:
this all looks great in my local test- only issue is the formatting may not be ideal. To fix, you would want to move the `</div>` on line 79 to line 98.

Reply:
current: 
![image](https://user-images.githubusercontent.com/1847818/206034717-cfc0fd5e-cdda-4461-863e-79382f6a49cc.png)


with above fix:
![image](https://user-images.githubusercontent.com/1847818/206034762-34e3903d-9612-4e18-b0b1-e5085bbe19aa.png)

Reply:
Thank you! Followed your instructions. Hope all is good now.